cves/CVE-2013-4444.yml

```diff
@@ -176,8 +165,11 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
```

Comment:
were there any commits you found between the fix and vcc? I recommend running the following: "git log --before={date when fix occured} --stat"